.github/workflows/linux-wpt.yml

```diff
@@ -87,25 +88,38 @@ jobs:
         with:
           name: wpt-full-logs-linux-${{ inputs.wpt-layout }}-${{ matrix.chunk_id }}
           path: wpt-full-logs/*/
+      - name: Archive wptreport
+        uses: actions/upload-artifact@v4
+        if: ${{ always() }}
+        with:
+          name: wptreport-linux-${{ inputs.wpt-layout }}-${{ matrix.chunk_id }}.json
+          path: wptreport/*/
```

Comment:
Let's incorporate this file into the full logs so that we still only have full and filtered. The reasoning here is two fold. Full/filtered/wptreport still raises the question: are the wptreport logs filtered? They aren't, so they should go with the full logs. The other thing is that I don't think it makes sense to create an artifact for every log type. We have a filtered version just because the full logs are larger and thus take longer to download. Thus it isn't a large concern to add another large file to this archive.

Reply:
In that case we need to introduce additional subfolders in zip so we can easier extract only wptreports/full logs.